ion/src/shared/platform_info.cpp

```diff
@@ -36,7 +36,7 @@ class PlatformInfo {
 #ifdef OMEGA_USERNAME
     m_username{OMEGA_USERNAME},
 #else
-    m_username{"\0"},
+    m_username{"\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"},
```

Comment:
This change isn't really needed but whatever

Reply:
This will be way easier for release (and if one day, there's a "upload your rom" solution in the installator, this will be needed :p

Reply:
Well since the size is defined here https://github.com/Omega-Numworks/Omega/blob/9eff895a95e17a0d5ad2e4894b37a6ccd4a72301/ion/src/shared/platform_info.cpp#L91 and the string is null-terminated (plus de C++ standard says an array is initialized with null bytes), it doesn't make a difference

Reply:
Ah yes. Forgot about it. Not a problem btw, so let's merge it :p